### DISCUSSION TITLE
Template Code

### DISCUSSION THREAD

**Discussion by DJ-CrossFade** [andyblac/UI-Minimalist-Custom-Cards#19] [Q&A] [answered]
This is the error I am getting from the room card:

Button-card template 'custom_card_andyblac_room' is missing!

I think it needs me to insert this template code somewhere, am I right?

```yaml title="card_room.yaml"
--8<-- "custom_cards/custom_card_andyblac_room/custom_card_andyblac_room.yaml"
```

My guess is I'm it needs to go here under template and replaces custom_card_andyblac_room maybe?

  - type: custom:button-card
    template:
      - custom_card_andyblac_room
    name: "Bathroom"
    tap_action:
      action: navigate

Can anyone help me?

**Comment by andyblac** [comment, marked as answer]
instructions on how to use, and a example of a config can be found here https://github.com/andyblac/UI-Minimalist-Custom-Cards/blob/main/docs/usage/custom_cards/custom_card_andyblac_room.md
